common/src/main/java/io/homeassistant/companion/android/common/data/integration/Entity.kt

```diff
@@ -700,70 +700,100 @@ val <T> Entity<T>.friendlyName: String
-        "fan_only" -> context.getString(commonR.string.state_fan_only)
-        "fog" -> context.getString(commonR.string.state_fog)
-        "hail" -> context.getString(commonR.string.state_hail)
-        "heat" -> context.getString(commonR.string.state_heat)
-        "heat_cool" -> context.getString(commonR.string.state_heat_cool)
-        "home" -> context.getString(commonR.string.state_home)
-        "idle" -> context.getString(commonR.string.state_idle)
-        "jammed" -> context.getString(commonR.string.state_jammed)
-        "lightning-raining" -> context.getString(commonR.string.state_lightning_raining)
-        "lightning" -> context.getString(commonR.string.state_lightning)
-        "locked" -> context.getString(commonR.string.state_locked)
-        "locking" -> context.getString(commonR.string.state_locking)
-        "mowing" -> context.getString(commonR.string.state_mowing)
-        "not_home" -> context.getString(commonR.string.state_not_home)
-        "off" -> context.getString(commonR.string.state_off)
-        "on" -> context.getString(commonR.string.state_on)
-        "open" -> context.getString(commonR.string.state_open)
-        "opening" -> context.getString(commonR.string.state_opening)
-        "partlycloudy" -> context.getString(commonR.string.state_partlycloudy)
-        "paused" -> context.getString(commonR.string.state_paused)
-        "pending" -> context.getString(commonR.string.state_pending)
-        "playing" -> context.getString(commonR.string.state_playing)
-        "problem" -> context.getString(commonR.string.state_problem)
-        "pouring" -> context.getString(commonR.string.state_pouring)
-        "rainy" -> context.getString(commonR.string.state_rainy)
-        "recording" -> context.getString(commonR.string.state_recording)
-        "returning" -> context.getString(commonR.string.state_returning)
-        "snowy-rainy" -> context.getString(commonR.string.state_snowy_rainy)
-        "snowy" -> context.getString(commonR.string.state_snowy)
-        "standby" -> context.getString(commonR.string.state_standby)
-        "streaming" -> context.getString(commonR.string.state_streaming)
-        "sunny" -> context.getString(commonR.string.state_sunny)
-        "triggered" -> context.getString(commonR.string.state_triggered)
-        "unavailable" -> context.getString(commonR.string.state_unavailable)
-        "unlocked" -> context.getString(commonR.string.state_unlocked)
-        "unlocking" -> context.getString(commonR.string.state_unlocking)
-        "unknown" -> context.getString(commonR.string.state_unknown)
-        "windy", "windy-variant" -> context.getString(commonR.string.state_windy)
-        else -> state
+    val attributes = this.attributes as Map<String, Any?>
+
+    var friendlyState = when (domain) {
+        // https://github.com/home-assistant/core/blob/dev/homeassistant/components/binary_sensor/strings.json#L113
+        "binary_sensor" -> {
```

Comment:
Switch these two around as the comment only applies to binary sensor (and else has the comment inside the block) 😇

```suggestion
        "binary_sensor" -> {
            // https://github.com/home-assistant/core/blob/dev/homeassistant/components/binary_sensor/strings.json#L113
```